### PR TITLE
Honour username when present in rsync_target

### PR DESCRIPTION
**Pull request by scollazo**

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -194,21 +194,31 @@
...
   delegate_to: "{{ archivematica_src_configure_dashboardsettings['url']|urlsplit('hostname') }}"
   remote_user: "{{ archivematica_src_configure_atom_ssh_user | default('artefactual') }}"
-  when: archivematica_src_configure_dashboardsettings is defined
+  when:
+    -  archivematica_src_configure_dashboardsettings is defined
 
 - name: "Add ssh key to AtoM server"
   authorized_key:
-    user: "archivematica"
+    user: "{{ atom_dipupload_ssh_user }}"
```
Comment (mamedin) [inline tasks/configure.yml]: The above line must be : 
```
user: {{ atom_dipupload_ssh_user | default('archivematica') }}
```

When using archivematica as default user (rsync_target: "atomdemo.artefactual.com:/tmp"), it fails with:

```
  TASK [artefactual.archivematica-qa-src : Add ssh key to AtoM server] ********************************************************************************************************************************************************************************
fatal: [mamedin-test-centos]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'atom_dipupload_ssh_user' is undefined\n\nThe error appears to have been in '/home/maml/artefactual/deployment-qa-servers/envs/qa-1.11/roles/artefactual.archivematica-qa-src/tasks/configure.yml': line 219, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"Add ssh key to AtoM server\"\n  ^ here\n"}
   to retry, use: --limit @/home/maml/artefactual/deployment-qa-servers/envs/qa-1.11/am-install.retry
```

**Review by mamedin — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 1 above)
Comment (scollazo) [reply tasks/configure.yml]: Good catch, fixed.

**Comment by scollazo** [comment]
Connects to https://github.com/archivematica/Issues/issues/931

**Review by mamedin — APPROVED** [review]
LGTM

**Pull request merged**